stories/guides/CustomView.stories.mdx

```diff
@@ -0,0 +1,116 @@
+import { Meta } from '@storybook/addon-docs'
+import LinkTo from '@storybook/addon-links/react'
+
+<Meta title="Guides/Creating Custom Views" />
+
+# Creating A Custom View
+
+It is possible to override an existing Calendar `view`, or even create a new `view` all together, by adhering to a specific interface.
+
+<LinkTo kind="examples" story="example-9">
+  View An Example
+</LinkTo>
+
+First build your custom `view` component. This could be based off an existing component, or something completely custom.
+
+```js
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Navigate as navigate } from 'react-big-calendar'
```

Comment:
`Navigate` is renamed to `navigate` but in the example it's used as `Navigate` (`Navigate.NEXT` and `Navigate.PREVIOUS`). Do you think the renaming can be removed?